Use commitments from config, fixes to issue and redeem for no-reload

durch opened the pull request:
@alxdavids Resubmitting our latest compatibility fixes, as per your suggestion we're using commitments from `config.js`, also fixing some issues we've found with the `no-reload` method.

Feedback is very welcome!

alxdavids commented on src/ext/background.js:
```diff
@@ -460,6 +460,9 @@ function beforeRequest(details, url) {
 
     // actually send the token signing request via xhr and return the xhr object
     const xhr = sendXhrSignReq(xhrInfo, url, details.tabId);
+    if (xhrInfo.cancel) {
```
I'm not sure exactly what this is being used to do?

alxdavids commented on src/ext/issuance.js:
```diff
@@ -62,15 +62,16 @@ function signReqCF(url) {
 /**
  * hCaptcha issuance request
  * @param {URL} url
+ * @param {DETAILS} details
```
nit: `{Object}`

durch commented:
@alxdavids thank you for the quick turnaround, fixed that silly definition error, also added a brief description of the issue workflow. I'd be happy to hop on a call and go over it in more detail :)
